types/events.go

```diff
@@ -165,6 +165,7 @@ var (
 	EventQueryNewEvidence         = QueryForEvent(EventNewEvidence)
 	EventQueryNewRound            = QueryForEvent(EventNewRound)
 	EventQueryNewRoundStep        = QueryForEvent(EventNewRoundStep)
+	EventQueryNewSignedBlock      = QueryForEvent(EventSignedBlock)
```

Comment:
[nit][question] for consistency with the other events in this list, should both the terms have `New` or neither? So

```go
	EventQueryNewSignedBlock      = QueryForEvent(EventNewSignedBlock)
```

or 

```go
	EventQuerySignedBlock      = QueryForEvent(EventSignedBlock)
```

Reply:
Yeah probably should be without the `New` but I think I will just pass on the nit as its already merged